src/docfx/config/Config.cs

```diff
@@ -166,10 +166,9 @@ internal sealed class Config
         public readonly SourceInfo<string>[] MetadataSchema = Array.Empty<SourceInfo<string>>();
 
         /// <summary>
-        /// Get the theme repo url like https://github.com/docs/theme#master
-        /// It's used for legacy doc(docs.com) sites build
+        /// Get the template folder or git repository url (like https://github.com/docs/theme#master)
         /// </summary>
-        public readonly string Theme = string.Empty;
+        public readonly string Template = string.Empty;
```

Comment:
that's a break change, also need change it Configuration Migration Tool, @mingwli

Reply:
yep, intentional